bundles/org.corpus_tools.hexatomic.corpusedit/src/main/java/org/eclipse/wb/swt/ResourceManager.java

```diff
@@ -1,14 +1,10 @@
 /*******************************************************************************
- * Copyright (c) 2011 Google, Inc. and others
- * All rights reserved. This program and the accompanying materials
- * are made available under the terms of the Eclipse Public License v1.0
- * which accompanies this distribution, and is available at
- * http://www.eclipse.org/legal/epl-v10.html
- *
- * Contributors:
- *    Google, Inc. - initial API and implementation
- *    Wim Jongman  - 1.8 and higher compliance
+ * Copyright (c) 2011 Google, Inc. and others All rights reserved. This program and the accompanying
+ * materials are made available under the terms of the Eclipse Public License v1.0 which accompanies
+ * this distribution, and is available at http://www.eclipse.org/legal/epl-v10.html Contributors:
```

Comment:
```suggestion
 * this distribution, and is available at http://www.eclipse.org/legal/epl-v10.html 
 *
 * <p>
 * Contributors:
```

Improve readability while keeping the style valid.